packages/eslint-plugin/lib/processors/use-jessie.js

```diff
@@ -9,80 +12,90 @@ const jessieRulesOneLine = `\
  curly: ['error', 'all']\
 ,eqeqeq: ['error', 'always']\
 ,no-bitwise: ['error']\
-,no-fallthrough: ['error', { commentPattern: 'fallthrough is not allowed' }]\
-,no-plusplus: ['error']\
+,no-fallthrough: ['error', { commentPattern: 'fallthrough is not allowed in Jessie' }]\
 ,no-restricted-globals: ['error', 'RegExp', 'Date']\
 ,no-restricted-syntax: ['error', \
 {\
   selector: "BinaryExpression[operator='in']",\
-  message: "'in' is not allowed.",\
+  message: "'in' is not allowed in Jessie",\
+},\
+{\
+  selector: "UpdateExpression[operator='++'][prefix=false]",\
+  message: "postfix '++' is not allowed in Jessie",\
+},\
+{\
+  selector: "UpdateExpression[operator='--'][prefix=false]",\
+  message: "postfix '--' is not allowed in Jessie",\
 },\
 {\
   selector: "BinaryExpression[operator='instanceof']",\
-  message: "'instanceof' is not allowed.",\
+  message: "'instanceof' is not allowed in Jessie",\
 },\
 {\
   selector: 'NewExpression',\
-  message: "'new' is not allowed.",\
+  message: "'new' is not allowed in Jessie",\
 },\
 {\
   selector: 'FunctionDeclaration[generator=true]',\
-  message: 'generators are not allowed.',\
+  message: 'generators are not allowed in Jessie',\
 },\
 {\
   selector: 'FunctionDeclaration[async=true]',\
-  message: 'async functions are not allowed.',\
+  message: 'async functions are not allowed in Jessie',\
 },\
 {\
   selector: 'FunctionExpression[async=true]',\
-  message: 'async functions are not allowed.',\
+  message: 'async functions are not allowed in Jessie',\
 },\
 {\
   selector: 'ArrowFunctionExpression[async=true]',\
-  message: 'async functions are not allowed.',\
+  message: 'async functions are not allowed in Jessie',\
 },\
```

Comment:
Does this set also catch async generator functions `async function* ...` ?

Reply:
Yes it does, it classifies them as `[generator=true][async=true]` which matches the `[generator=true]` rule.

However, we probably need some refinement to accept our function-level `await` rules.  This will later need some work.

Reply:
> However, we probably need some refinement to accept our function-level await rules. This will later need some work.

Good point!